bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -147,8 +147,13 @@ async def async_run_method(
 
         multipart = payload_params_to_multipart(payload_params)
         client = self._get_client()
+
+        method_name = __bentoml_method.name
+        if __bentoml_method.name == "__call__":
+            method_name = ""
+
         async with client.post(
-            f"{self._addr}/{__bentoml_method.name}",
+            f"{self._addr}/{method_name}",
```

Comment:
could we handle both "/__call__" and "/" on the server side? @bojiang @larme

Reply:
Then what if there was a `call` on the runnable?

Reply:
@bojiang oh I mean `/__call__`

Reply:
@parano ah I agree. Will open a PR for that